realcc_tests/test_procedure.py

```diff
@@ -183,6 +183,14 @@ def run(self) -> bool:
     ),
 ]
 
+DASHCAST_CTRL_TESTS = [
+    CattTest(
+        "cast GitHub website frontpage",
+        ["cast_site", "https://github.com"],
+        check_data=("status_text", "The world’s leading software development platform · GitHub"),
```

Comment:
This might be a bit too specific, can we check that only `GitHub` is in the data? We don't want tests breaking if they change anything about their tagline...

Reply:
Fixed.